pktoken/pktoken.go

```diff
@@ -175,11 +184,22 @@ func (p *PKToken) ProviderAlgorithm() (jwa.SignatureAlgorithm, bool) {
 	return alg.(jwa.SignatureAlgorithm), true
 }
 
+// Deprecated: The PK Token now stores the signed tokens such as OpToken,
```

Comment:
If this PR is deprecating a function, it seems like we should be replacing everywhere we internally use that function in this PR, otherwise I'd doubt it would ever be done and therefore we would never be able to remove the function

Reply:
Next PR after this one removes Compact. I wanted to keep this PR small and fast to review so I'm adding the bugfix here and them removing it later.

I created an issue to track this https://github.com/openpubkey/openpubkey/issues/107